extensions/vscode/src/utils/window.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2025 by Posit Software, PBC.
+
+import { window } from "vscode";
+
+export function showErrorMessageWithTroubleshoot(
```

Comment:
This is a nice approach! wrapping the vscode usage here helps on mocking this module where `showErrorMessageWithTroubleshoot` is used for tests.

We could definitely do the same for other messaging methods we have on the code base.